### DISCUSSION TITLE
Share Workspace

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [codesandbox/codesandbox-client#6505] [Q&A] [answered]
Hey there, :)

I know this question might be a bit silly, obvious, and unnecessary, or I'm being a blind noob,  but there is any way to share with someone my workspace?

I don't want the person to have access to edit the sandboxes, I just want to show what I've been doing, this is my user:
https://codesandbox.io/u/201flaviosilva

And I'm in a workspace called "Impacto", and I want to share the link of workspace :)

(Sorry, for my English :))

<img width="233" alt="Captura de ecrã 2022-03-01, às 12 48 52" src="https://user-images.githubusercontent.com/55815265/156171805-cf8067f1-6baf-43b8-9e5c-d538d4e6df9a.png">

**Comment by JamesACS** [comment, marked as answer]
Hey @201flaviosilva - Absolutely !

Just open the Sandbox you want to share and click the 'two people' icon on the left hand bar to get your sharing link!

**Reply by 201flaviosilva** [reply]
Hi, thanks a lot for the answer, but that's not what I want, I would like to be able to share a complete workspace, not just a project.

For example, I created a workspace in my account, called "impact", my goal is to get a link, so I can share all the sandboxes I made with that workspace. As GitHub has the organizations :)

Am I making sense?

**Reply by 201flaviosilva** [reply]
I'm creating a simple framework and I would like to be able to share all the examples I make of the framework from a workspace, so that people can enter the link and have access (only as a viewer) to all the sandboxes :)

I feel like maybe I'm being confused? 😅  @JamesACS  :)

**Reply by JamesACS** [reply]
Ah @201flaviosilva - In this case you'd need to create a team under our Pro plan! You can then add as many people as you like as viewers to your Team projects without extra cost (just make sure not to make them administrators or editors)!

**Reply by 201flaviosilva** [reply]
Ah ok thank you very much for the answer, I will consider the solution :)
